dev-rust/cbindgen/cbindgen-0.26.0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A tool for generating C binding to Rust code"
+DESCRIPTION="cbindgen creates C/C++11 headers \
+for Rust libraries which expose a public C API."
+HOMEPAGE="https://github.com/eqrion/cbindgen"
+COPYRIGHT="2018-2023 cbindgen contributors"
+LICENSE="MPL v2.0"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="b45e1a64875b615702a86ac3084ef69ae32926241cd2b687a30c12474be15105"
+SOURCE_FILENAME="cbindgen-$portVersion.tar.gz"
+
+ARCHITECTURES="all !x86_gcc2"
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	cbindgen$secondaryArchSuffix = $portVersion
+	cmd:cbindgen
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cargo$secondaryArchSuffix >= 0.70
+	cmd:gcc$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage cbindgen$secondaryArchSuffix \
+	"$commandBinDir"/cbindgen
+
+BUILD()
+{
+	cargo fetch --locked
```

Comment:
shouldn't it specify --target ? instead of all architectures.

Reply:
Arch uses: ``cargo fetch --locked --target "$(rustc -vV | sed -n 's/host: //p')"`` . Don't know if it works on Haiku.

Reply:
korli's suggested command seems to work
```
~/src/gecko-dev> rustc -vV | sed -n 's/host: //p'
x86_64-unknown-haiku
```

Reply:
Will run some checks on this here, got/am side-tracked again. :)

Reply:
Change applied, haven't checked 32bit yet (won't be for today).